test/functional/apps/console/_text_input.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const retry = getService('retry');
+  const log = getService('log');
+  const PageObjects = getPageObjects(['common', 'console', 'header']);
+  const toasts = getService('toasts');
+
+  describe('text input', function testTextInput() {
+    this.tags('includeFirefox');
+    before(async () => {
+      log.debug('navigateTo console');
+      await PageObjects.common.navigateToApp('console');
+      await PageObjects.console.closeHelpIfExists();
+    });
+
+    describe('load_from parameter', () => {
+      it('should load the data from url', async () => {
+        await PageObjects.common.navigateToApp('console', {
+          hash: '#/console?load_from=https://www.elastic.co/guide/en/elasticsearch/reference/current/snippets/2199.console',
+        });
+        await retry.try(async () => {
+          const request = await PageObjects.console.getRequest();
+          log.debug(request);
+          expect(request.trim()).to.eql('PUT /my-index-000001');
+        });
+      });
+
+      it('should load the data from the data URI', async () => {
+        await PageObjects.common.navigateToApp('console', {
+          hash: '#/console?load_from=data:text/plain,BYUwNmD2Q',
+        });
+
+        await retry.try(async () => {
```

Comment:
nit: I can recommend you to move `retry.try` inside the page object (`PageObjects.console.getRequest`) .  It will make your code cleaner